Add 1kb.tham.es

richardthames opened the pull request:
The [GTmetrix Performance Report](https://gtmetrix.com/reports/1kb.tham.es/3dhBmmfR/) indicates .99KB (not very exact).

This is sad, because the source is _exactly_ a cool 777B.

I think this might be one of the first with a favicon.

richardthames committed:
Add 1kb.tham.es

bradleytaunt commented:
You need to include the proper `md` file extension. Also I am consistently getting 1000 bytes: https://gtmetrix.com/reports/1kb.tham.es/GWWGuXK0/

richardthames committed:
Merge branch 'bradleytaunt:master' into master

richardthames committed:
Update 1kb.tham.es

richardthames commented:
I added the `md` file extension, and updated the size based on optimizations and a [new GTmetrix Performance Report](https://gtmetrix.com/reports/1kb.tham.es/x9oxNXYz/).
